packages/cli/src/commands/serve.ts

```diff
@@ -6,26 +6,39 @@ import { viteExternalsPlugin } from 'vite-plugin-externals';
 import { Logger } from '../utils/logger.js';
 import { getAppBridgeVersion } from '../utils/appBridgeVersion.js';
 import pkg from '../../package.json';
+import path from 'path';
+import { fileURLToPath } from 'url';
 
 class PlatformAppDevelopmentServer {
     constructor(private readonly port: number) {}
 
     async serve(): Promise<void> {
+        Logger.info('Development');
+
+        const __dirname = path.dirname(fileURLToPath(import.meta.url));
         try {
             const viteServer = await createServer({
                 root: process.cwd(),
-                configFile: false,
```

Comment:
Removing this would allow external devs to use `vite.config.ts`, I am more inclined to have a `frontify.config.ts` which we control